e2e/tests/user/registration.spec.ts

```diff
@@ -54,16 +55,22 @@ describe('User registration tests:', () => {
 
     await createAccountPage.acceptTermsOfUseAgreement();
     let nextButton = await createAccountPage.getNextButton();
+    await nextButton.waitUntilEnabled();
```

Comment:
Can we add comments here to // Step 2: Terms of service?

Reply:
has a comment, it is at line #51.